packages/blade/.babelrc.js

```diff
@@ -82,7 +82,6 @@ const configs = {
         [
           '@babel/plugin-transform-runtime',
           {
-            corejs: 3,
```

Comment:
this was the culprit. The plugin defaults to assuming that all polyfillable APIs will be provided by the user. Otherwise the [corejs](https://babeljs.io/docs/en/babel-plugin-transform-runtime#corejs) option needs to be specified. Specifying a number will rewrite the helpers that need polyfillable APIs to reference helpers from that (major) version of core-js instead and then consumers need to install that. can't think of a use case yet where this will be helpful even the bundlesize doesn't gets impacted. infact after removing this we reduced good 7-8kbs of bundlesize 😄 

[More info on babel's site](https://babeljs.io/docs/en/babel-plugin-transform-runtime#options)

Reply:
This doesn't affect react native right?

Reply:
> This doesn't affect react native right?

nupe for the first time in history 😆